build-system/tasks/helpers.js

```diff
@@ -729,7 +729,7 @@ function massageSourcemaps(sourcemaps, babelMaps, options) {
   );
 
   remapped.sources = remapped.sources.map((source) => {
-    if (source.startsWith('/__SOURCE__/')) {
+    if (source?.startsWith('/__SOURCE__/')) {
       return source.slice('/__SOURCE__/'.length);
     }
     return source;
```

Comment:
Purely stylistic suggestion since I find ternaries clearer in these cases, feel free to ignore
```suggestion
    return source?.startsWith('/__SOURCE__/'))
      ? source.slice('/__SOURCE__/'.length)
      : source;
```

Also, because I wanted to see what it looks like:
```javascript
return source.match(/(?<=^\/__SOURCE__\/).*$/)?.[0] ?? source
```